docs/spec/wrapper-sanitations.md

```diff
@@ -1,4 +1,4 @@
-# Sanitizations for client
+# Sanitations for client
```

Comment:
Lets use the word `sanitization` to be consistent with the other connector docs https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/pull/264/files#diff-e97a89afc8ae756582374bbfa7066911897ce0106567dbdf4f23cf6ec3ba5a33

Reply:
Changed with [ee499ad](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/229/commits/ee499adf680b027f71751549560724dd5e09a396).